Rubberduck.CodeAnalysis/Inspections/Concrete/ExcessiveInterfaceMembersInspection

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Linq;
+using Rubberduck.CodeAnalysis.Inspections.Abstract;
+using Rubberduck.Common;
+using Rubberduck.Parsing.Annotations;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.Parsing.VBA.DeclarationCaching;
+using Rubberduck.Resources.Inspections;
+
+namespace Rubberduck.CodeAnalysis.Inspections.Concrete {
```

Comment:
Right, I did notice that. I use the other style because it is easier to read for me but I will make sure to convert back before I commit anything else.